### ISSUE TITLE
Build error: Zone js wih @ngtools/webpack `1.10.1`

### ISSUE THREAD

**Issue by alan-agius4** [angular/angular-cli#9727]
### Versions
```
@ngtools/webpack: 1.10.1
```

I am using the `@ngtools/webpack` outside of the Angular CLI, and the the latest patch version from 1.10.0 to 1.10.1. I am getting the following error;

```bash
Module not found: Error: Can't resolve 'C:\git\solutions\webpack\buildin\global.js' in 'C:\git\solutions\site\node_modules\zone.js\dist'
```

### Observed behavior
```
It looks the the module resolution or webpack context is not correct.
```

### Desired behavior
No error.

### Mention any other details that might be useful (optional)
I am thinking it might be related to this: https://github.com/angular/angular-cli/commit/595168eeb251c9928d42139424e05dcb50270b08#diff-c8ee7673e7d5e8741306839a34f0c8fc

I'll try to debug some more.

**Comment by filipesilva** [comment]
This seems like a bug but we'll need to look at a reproduction to find and fix the problem. Can you setup a minimal repro please? 

You can read [here](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) why this is needed. A good way to make a minimal repro is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Then you can push this repository to github and link it here.

**Comment by alan-agius4** [comment]
Yeah, I’ll try to submit a minimal repo. Cause the thing is that I am not using Angular CLI. I’ll do my best to try to replicate it.

**Comment by alan-agius4** [comment]
Hey @filipesilva,

I managed to replicate this and a reproduction can be found here.
https://github.com/alan-agius4/angular-cli-issue-9727

What I did was to add `paths` in  a `tsconfig` resolution.
https://github.com/alan-agius4/angular-cli-issue-9727/blob/master/src/tsconfig.app.json#L9

Something like this was working until @ngtools/webpack 1.10.1
